styles/fonts.less

```diff
@@ -127,6 +127,10 @@
 .font ( 'Monoid', bold, normal, 'monoid/monoid-bold.ttf' );
 .font ( 'Monoid', normal, italic, 'monoid/monoid-italic.ttf' );
 .font ( 'Monoid', normal, normal, 'monoid/monoid.ttf' );
+.font ( 'Mononoki', bold, normal, 'mononoki/mononoki-Bold.ttf' );
+.font ( 'Mononoki', bold, italic, 'mononoki/mononoki-BoldItalic.ttf' );
```

Comment:
`-Bold` etc. should be lowercase and `-bold-italic` needs to have a dash in between. I need the consistency to be able to reuse this across projects.